### PR TITLE
Changed class name for manage packages extension

### LINKED ISSUE #8721: Notebooks: Contributed Actions on Toolbar have Different Text Color on Dark Theme
image

Notice how the Manage Packages action has text that's brighter than the Collapse Cells action.

### PR DESCRIPTION
**Pull request by smartguest**
Since the manage packages extension class name is .icon instead of .codicon, the style rules need to account for this as well (changing the class name to .codicon breaks the icon image so its not recommended)
This PR fixes #8721

### CODE REVIEW

**Comment by anthonydresser** [comment]
Probably just makes more sense to update the extension, should be a bug in the extension not core code.

**Comment by smartguest** [comment]
I figured, this would make it inconsistent

**Comment by smartguest** [comment]
Code now has the fixes put in the right place, also the icon class name has been renamed and placed after the other parts of the class name for consistency.

File: src/vs/platform/actions/browser/menuEntryActionViewItem.ts [hunk 1]
```diff
@@ -262,17 +262,17 @@ export class MenuEntryActionViewItem extends ActionViewItem {
 					iconClass = MenuEntryActionViewItem.ICON_PATH_TO_CSS_RULES.get(iconPathMapKey)!;
 				} else {
 					iconClass = ids.nextId();
-					createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
-					createCSSRule(`.vs-dark .icon.${iconClass}, .hc-black .icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.dark)}`);
+					createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
```
Comment (Charles-Gagnon) [inline src/vs/platform/actions/browser/menuEntryActionViewItem.ts]: If you're making changes to core VS code you should be adding a SQL CARBON EDIT to the affected lines

But I'm not sure about making the change here - do you have a clear idea of what side effects this might cause? I dug through the code a little bit and it doesn't seem like it's used outside the debug viewlet but I might have missed something. We should be careful making core changes like this in case we break other things in the process.

I tested just making the codicon changes to the LabeledMenuItemActionItem class below and that seems to fix the issue and is much more specific to this case (the task bar instead of a general Menu item). I think it'd be better to revert the changes to this base class and instead just make them in the LabeledMenuItemActionItem class.

Comment (smartguest) [reply src/vs/platform/actions/browser/menuEntryActionViewItem.ts]: Alright I'll revert the ones for the core classes, but searching around the code, I did not see any rules for .icon so far, and it didn't break anything when I tested it, also I'll add the SQL CARBON EDIT as well.

File: src/vs/platform/actions/browser/menuEntryActionViewItem.ts [hunk 2]
```diff
@@ -333,13 +335,13 @@ export class LabeledMenuItemActionItem extends MenuEntryActionViewItem {
 				iconClass = MenuEntryActionViewItem.ICON_PATH_TO_CSS_RULES.get(iconPathMapKey)!;
 			} else {
 				iconClass = ids.nextId();
-				createCSSRule(`.icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
-				createCSSRule(`.vs-dark .icon.${iconClass}, .hc-black .icon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.dark)}`);
+				createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
```
Comment (Charles-Gagnon) [inline src/vs/platform/actions/browser/menuEntryActionViewItem.ts]: I was suggesting that you modify the LabeledMenuItemActionItem class since that's a custom class we added ourselves and is what's used for the action bar. The less we modify core VS code the better. 

I would also revert the change to the ids unless it's actually necessary - again so that we reduce the number of conflicting changes we make to core VS code.

Comment (smartguest) [reply src/vs/platform/actions/browser/menuEntryActionViewItem.ts]: Changed the id back to the old one.

File: src/vs/platform/actions/browser/menuEntryActionViewItem.ts [hunk 3]
```diff
@@ -135,6 +135,8 @@ export function createActionViewItem(action: IAction, keybindingService: IKeybin
 
 const ids = new IdGenerator('menu-item-action-item-icon-');
 
+// {{SQL CARBON EDIT}} - This is here to use the 'ids' generator above
```
Comment (Charles-Gagnon) [inline src/vs/platform/actions/browser/menuEntryActionViewItem.ts]: Remove these comments since we're not changing any of this code anymore - less conflicts with the VS stuff
